client/src/plugins/usage-statistics/event-handlers/__tests__/DiagramOpenEventHandlerSpec.js

```diff
@@ -1159,6 +1206,68 @@ describe('<DiagramOpenEventHandler>', () => {
       });
     });
 
+    it('should send Platform engine profile (forms)', async () => {
+
+      // given
+      const subscribe = sinon.spy();
+
+      const onSend = sinon.spy();
+
+      const tab = createTab({
+        type: 'form',
+        schema: engineProfilePlatform
+      });
```

Comment:
Please keep the tab type as is. The tab should have a file property which has contents of type String. So my tab can look like:
```js
const tab = {
  file: {
    contents: '{"key": "value"}'
}
```

Reply:
Also, let's use real-life Camunda Forms files here.